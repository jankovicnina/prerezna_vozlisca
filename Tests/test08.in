42 55
15 33 1.92
25 34 -1.38
1 7 6.37
34 26 7.62
13 3 8.47
19 41 3.11
26 9 2.98
9 17 8.64
31 24 4.41
29 17 4.22
41 31 9.08
12 29 1.89
20 25 7.56
21 9 8.78
23 6 2.02
11 39 6.33
18 19 8.55
27 5 4.21
17 32 8.1
15 40 7.46
35 27 7.94
30 2 3.51
28 8 8.11
3 20 -0.94
7 22 -1.59
24 14 7.54
21 35 4.28
33 20 6.01
14 5 4.38
22 4 9.61
31 12 5.32
2 12 3.3
27 1 1.61
39 30 5.13
3 36 1.08
32 20 8.02
36 37 10.0
8 21 2.33
4 10 6.92
6 28 3.41
2 15 -1.96
32 28 9.07
36 35 2.54
38 33 1.86
30 1 -1.45
10 0 9.97
5 23 5.9
16 10 5.73
26 41 5.74
37 38 3.38
6 11 3.62
16 24 0.8
29 13 7.69
0 16 2.3
40 18 7.34
