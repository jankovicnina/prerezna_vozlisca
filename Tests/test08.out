44.28236353077824
